brands/amenity/fast_food.json

```diff
@@ -676,6 +669,18 @@
       "takeaway": "yes"
     }
   },
+  "amenity/fast_food|Donut King": {
+    "matchNames": ["donut king"],
```

Comment:
if this is also only in Australia can you add the country code so we don't show it world wide